components/LeagueEntries/LeagueEntries.tsx

```diff
@@ -107,14 +143,22 @@ const LeagueEntries = ({
           data-testid="league-entry-pick-button-container"
         >
           {!isEliminated && (
-            <Link href={linkUrl} data-testid="league-entry-pick-button-link">
-              <Button
-                className="league-entry-pick-button"
-                data-testid="league-entry-pick-button"
-                label={isPickSet ? 'Change Pick' : 'Make Pick'}
-                variant={isPickSet ? 'secondary' : 'default'}
-              />
-            </Link>
+            <LinkCustom
+              aria-disabled={isLockedOut === true ? 'true' : 'false'}
+              className={cn(
+                'league-entry-pick-link',
+                isLockedOut === true ? 'opacity-50 cursor-not-allowed' : '',
+              )}
+              dataTestidProp="league-entry-pick-link"
+              href={linkUrl}
+              onClick={(e: { preventDefault: () => unknown }) =>
+                isLockedOut === true && e.preventDefault()
+              }
+              size={'defaultButton'}
+              variant={isPickSet ? 'secondaryButton' : 'primaryButton'}
```

Comment:
Q: Here you use truthy/falsy value, but above with isLockedOut, you explicitly compare it with the `true` boolean value. How come? This seems inconsistent.

Reply:
@ryandotfurrer this as well

Reply:
@alexappleget did you get guidance on this? 
can be rewritten as `isLockedOut && e.preventDefault()` like in line 158
In a ternary operator, it is evaluating the condition before the '?' . isLockedOut is a boolean value, so it counts as an expression without the `=== true`

Reply:
There's 2 separate convos going on here. The line for comment is line 131, variants. Y'all are also talking about `aria-disabled` attribute. We can just use `lockedOut` because it's a boolean already. For the variants, this isn't tied to the lockdown period directly. It's when a user has a pick set or not.

Reply:
@shashilo ty for clarifying for everyone. 

@choir241 if this is OK, please resolve this conversation.